### PR TITLE
fixing memento

### PR DESCRIPTION
**Pull request by olhabn**

### CODE REVIEW

**Review by jlhughes — APPROVED** [review]
Tested, this fixes the issue with multipart keys.

**Comment by olhabn** [comment]
only memento fix. adding multipart key support

File: src/foam/u2/view/ScrollTableView.js [hunk 1]
```diff
@@ -224,7 +224,18 @@
 
         var id = mementoHead;
         if ( of ) {
-          id = of.ID.fromString(mementoHead);
+          if ( ! foam.core.MultiPartID.isInstance(of.ID) ) {
+            id = of.ID.fromString(mementoHead);
+          } else {
```
Comment (olhabn) [inline src/foam/u2/view/ScrollTableView.js:229]: so probably ugly approach but I don't have other options.
if id is object (for MultiPartID) we will get rid of '{', '}', ':' as if we will let them stay then there is no difference between head and parameters. so head will look like `name="myAlarm",hostname="admins-MacBook-Pro"`

**Comment by kgrgreer** [comment]
I'm merging this as a temporary solution until @olhabn can implement the alternate design which we've discussed.

**Pull request merged**